docs/sources/plugins/developing/dataframe.md

```diff
@@ -60,6 +60,26 @@ Each field has three values, and each value in a field must share the same type.
 
 One restriction on data frames is that all fields in the frame must be of the same length to be a valid data frame.
 
+#### Field configuration
+
+Each field in a data frame contains optional information about the values in the field, such as units, scaling, and more.
+
+By adding field configurations to a data frame, Grafana can configure visualizations automatically, by for example setting the unit provided by the data source.
+
+### Transformations
+
+Along with the type information, field configs enables _data transformations_ within Grafana.
+
+A data transformation is any function that accepts a data frame as input, and returns another data frame as output.
+
+Here's a few examples of data frame transformations:
```

Comment:
Transformations are different from backend GEL (maybe renamed to pipelines, TBD). I think this all reads fine, but might be saying you can do things that you can't do yet - not entirely sure.

Reply:
As I understand it, GEL is a _language_ for _expressing_ transformations? 

Can you give 3(?) examples of data frame transformations that you'll be able to do in 7.0?

Reply:
GEL can transform data, but Transformations and (currently-named) GEL are different features. 

I don't really have good examples :-/ 

 1. Frontend transforms - I'm not familiar with the feature yet
 2. GEL - just picking it up again for the first time this calendar year, so what it can do for 7.0 - I don't really exactly, or even how to succinctly describe what it can do now. 

Right now GEL can, when Mars is in the correct position, do stuff with time series responses from backend plugins, such as:
 - Resample them
 - Reduce them
 - Do basic arithmetic on them 
 - Combine the above

FWIW, here are a couple screenshots:

![image](https://user-images.githubusercontent.com/1692624/80530752-78d3ba80-8967-11ea-9c7f-97b107464345.png)

![image](https://user-images.githubusercontent.com/1692624/80530819-94d75c00-8967-11ea-8d70-7ff8c75d99f7.png)

Reply:
I removed the examples for now. I think Hemingway (probably) said something about "incomplete docs is better than incorrect docs".